test/bdd/pkg/issuer/issuer_steps.go

```diff
@@ -254,6 +259,12 @@ func (e *Steps) signCredential(didDocVar string) ([]byte, error) {
 	return responseBytes, nil
 }
 
+func (e *Steps) issueCred(user string) error {
```

Comment:
It is funny that issueCred calls verifyCredential only

Reply:
I'll update in next PR.